crt_portal/static/sass/custom/landing.scss

```diff
@@ -18,6 +18,18 @@ $arrow-height: 24px;
   li {
     margin-bottom: 1rem;
   }
+
+  @include at-media-max(mobile-lg) {
+      padding-top: 3rem;
+      padding-bottom: 3rem;
```

Comment:
Looking at these rules more closely, I think we can simplify this. After looking at the [desktop design files](https://app.zenhub.com/workspaces/doj-crt-form-design-flows-5d7a67fa86a8cd0001581ea0/issues/18f/crt-django/79), I think lines 4 & 5 of this file should be changed to `3rem` so that the spacing on desktop bumps from 64px to 72px to match the mock. The padding rules on lines 23 & 24 can then be deleted.